File: Cmdlets/Models/BankExchange.vb
Imports TIKSN.Finance

Public Class BankExchange

	Public Sub New(BankName As String, Exchange As ICurrencyConverter)
		Me.BankName = BankName
		Me.Exchange = Exchange
	End Sub

	Public ReadOnly Property BankName As String

	Public ReadOnly Property Exchange As ICurrencyConverter

End Class

File: C#/Cmdlets/Cmdlets/Models/BankExchange.cs
using TIKSN.Finance;

namespace TIKSN.Cmdlets.Models
{
    public class BankExchange
    {
        public BankExchange(string BankName, ICurrencyConverter Exchange)
        {
            this.BankName = BankName;
            this.Exchange = Exchange;
        }

        public string BankName { get; }

        public ICurrencyConverter Exchange { get; }
    }
}